### DISCUSSION TITLE
crc start: A new version (1.34.0) has been published

### DISCUSSION THREAD

**Discussion by syedriko** [okd-project/okd#971] [Q&A] [answered]
The latest OKD crc from https://dl.fedoraproject.org/pub/alt/okd-crc/release/linux-amd64/ prints out 
```
$ crc start
WARN A new version (1.34.0) has been published on https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/1.34.0/crc-linux-amd64.tar.xz 
```

This is confusing because it refers to the OCP version of CRC. It'd be better if it checked for a newer OKD CRC version or not check at all.

**Comment by vrutkovs** [comment, marked as answer]
Right, we need to rip it out of OKD version. @cgruver to the rescue!

**Comment by cgruver** [comment]
I'll take a look at stripping that message out.
